python/ql/src/semmle/python/regex.qll

```diff
@@ -67,8 +108,8 @@ string mode_from_mode_object(Value obj) {
     result = "VERBOSE"
   ) and
   exists(int flag |
-    flag = Value::named("sre_constants.SRE_FLAG_" + result).(ObjectInternal).intValue() and
-    obj.(ObjectInternal).intValue().bitAnd(flag) = flag
+    flag = Value::named("sre_constants.SRE_FLAG_" + result).(OI::ObjectInternal).intValue() and
+    obj.(OI::ObjectInternal).intValue().bitAnd(flag) = flag
   )
```

Comment:
As far as I can tell, this does not involve points-to, but please reassure me that this is not code we will be moving away from :-)

Reply:
Your statement confuses me. The change you have highlighted above _does_ indeed involve points-to, and it _is_ code we will be moving away from. However, it's a predicate that _could_ have seen use outside of this codebase, and so I couldn't simply delete it.

I should add a comment saying it's deprecated, though, to make this clear.

Reply:
I have added the indicated statement. (And dated it, so we can actually see when we can delete it. :slightly_smiling_face:)

Reply:
Thanks, and I guess that the signature makes it rather awkward to rewrite it to not use points-to...